Classes/PackageFactory/AtomicFusion/Forms/Domain/Service/Runtime/Task/FinishTaskInterface.php

```diff
@@ -12,21 +12,29 @@
  */
 
 use Neos\Flow\Annotations as Flow;
-use Neos\Error\Messages\Result;
-use PackageFactory\AtomicFusion\Forms\Domain\Service\Runtime\FinisherRuntimeInterface;
+use PackageFactory\AtomicFusion\Forms\Domain\Service\Runtime\FormRuntimeInterface;
 use Neos\Flow\Http\Response;
 
 /**
- * Method definitions for the finish task
+ * Method definitions for generic Form Tasks
  */
 interface FinishTaskInterface
```

Comment:
Because the finisher tasks need the parenRequest. To further generalize this all tasks would have to get the runtime and the parentRequest.